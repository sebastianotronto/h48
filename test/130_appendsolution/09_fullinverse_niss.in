(F)
0
1
rotation UF
0
