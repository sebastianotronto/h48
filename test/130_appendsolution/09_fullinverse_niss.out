(F)
Number of solutions: 1
Shortest solution length: 1
Used bytes: 4
